source/adios2/helper/adiosDynamicBinder.cpp

```diff
@@ -46,7 +46,6 @@ DynamicBinder::DynamicBinder(std::string libName, std::string libPath)
 #endif
 
     std::vector<std::string> libSuffixes;
-    libSuffixes.emplace_back("");
```

Comment:
So this shouldn't be removed. It's supposed to be possible that you could tell ADIOS the full library name (e.g. `libMyPlugin.so` instead of just `MyPlugin`), so without this line ADIOS would fail in this situation.

Reply:
Oh I see, that makes sense